x-pack/plugins/apm/public/services/rest/index_pattern.ts

```diff
@@ -13,10 +13,3 @@ export const createStaticIndexPattern = async () => {
     signal: null,
   });
 };
-
-export const getApmIndexPatternTitle = async () => {
-  return await callApmApi({
-    endpoint: 'GET /api/apm/index_pattern/title',
-    signal: null,
-  });
-};
```

Comment:
This was unused afaict. Anyone know why we had it?

Reply:
Looks like we previously used it in WatcherFlyout: https://github.com/elastic/kibana/blob/16ba937bae63c9925b80285ad076325ee1a63569/x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/WatcherFlyout.tsx#L190

Reply:
Good find. Now I feel better about removing it.